%YAML:1.0

mode: 0

detector: 0

optimizer: 0

# ==================== camera informations ====================
width: 640
height: 480
# camera matrix
K: !!opencv-matrix
    rows: 3
    cols: 3
    dt: f
    data: [ 554.3827128226441, 0.0, 320.5, 0.0, 554.3827128226441, 240.5, 0.0, 0.0, 1.0 ]

# distortion coefficients
D: !!opencv-matrix
    rows: 1
    cols: 5
    dt: f
    data: [ 0.0, 0.0, 0.0, 0.0, 0.0 ]


camera_frame_id: "camera"

# ==================== marker informations ====================
# marker informations
marker_size: 0.079

rotation_vector: !!opencv-matrix
    rows: 1
    cols: 3
    dt: f
    data: [0.0, 0.0, 1.0]

plane_vector_1: 1
plane_vector_2: 3

# ==================== ros informations ====================
topic_image_aruco: "aruco_image"
topic_image_raw: "/camera/image_raw"

fps: 100

window_size: 10

P0: 10.0
Q: 1.0
R: 1.0